packages/live-preview-sdk/src/inspectorMode/utils.ts

```diff
@@ -239,7 +243,15 @@ export function getAllTaggedElements(root = window.document, ignoreManual?: bool
     taggedElements.push(element);
   }
 
-  return taggedElements;
+  const autoTaggedCount = Array.from(manualTagged).filter(
+    (el) => !el.hasAttribute(InspectorModeDataAttributes.MANUALLY_TAGGED),
+  ).length;
+
+  return {
+    taggedElements,
+    manuallyTaggedCount: manualTagged.length - autoTaggedCount,
```

Comment:
Umm.. why not manualTagged.length?

Reply:
I think I should rename `manualTagged` is it isn't just manual tagged. It's actually simply `tagged` as it just picks up the elements with the tagging data attributes. (was like this before the PR)

The logic is confusing I know, I'll tidy it up and add comments. But to clarify this is how it is now

- manualTagged (all elements that have tagging data attributes)
- uniqElementsForTagging are the elements that we have just added the autotagging to
- autoTaggedCount = manualTagged - the manualTagged that don't have the manual data attribute

Reply:
I've simplified the code thanks for calling it out